CODEOWNERS: Change the owner of drt and operations to DRP Eng

nameisbhaskar opened the pull request:
This changes the owner of the operations and drt* to DRP team

Epic: None
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/138186)
<!-- Reviewable:end -->

nameisbhaskar committed:
CODEOWNERS: Change the owner of drt and operations to DRP Eng

This changes the owner of the operations and drt* to DRP team

Epic: None
Release note:None

nameisbhaskar commented:
Thanks @vidit-bhat for your review.

srosenberg reviewed (approved):
Thanks!

nameisbhaskar commented:
Thanks @srosenberg for reviewing the change!

nameisbhaskar commented:
bors r=@vidit-bhat,@srosenberg
